tests/test_process_thermal.py

```diff
@@ -0,0 +1,40 @@
+import shutil
+import unittest
+from pathlib import Path
+
+from thermo_scenes.flir_thermal_images.custom_flir import CustomFlir
+
+
+class TestThermalConversion(unittest.TestCase):
+    def test_process_thermal(self) -> None:
+        """
+        Test of extracting RGB, thermal and csv files from the given msx images.
+
+        2 msx images will generate 2 RGB PNG files, 2 thermal PNG files, 2 csv files
+        and 1 temperature_bound.json file.
+        """
+        msx_path = Path("tests/data/process_thermal/msx")
+        output_path = Path("tests/data/process_thermal/output")
+
+        shutil.rmtree(output_path)
+        output_path.mkdir(exist_ok=True, parents=True)
+
+        CustomFlir(path_to_msx_images=msx_path, path_to_output_folder=output_path)
+
+        json_count = 0
+        PNG_count = 0
+        csv_count = 0
+
+        for folder in output_path.iterdir():
+            if folder.suffix == ".json":
+                json_count += 1
+            else:
```

Comment:
Instad of a if/else use continue after the if to skip the rest of the code and remove the else